src/cljs/rems/administration/workflow.cljs

```diff
@@ -3,11 +3,12 @@
             [re-frame.core :as rf]
             [rems.administration.administration :refer [administration-navigator-container]]
             [rems.administration.components :refer [inline-info-field]]
+            [rems.administration.resource :refer [licenses-view]]
```

Comment:
[nit] hmm maybe the license-view should be in rems.administration.license or rems.administration.components? resource is a funny place for it

Reply:
Done.